blank_rust_project/contract/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "status-message"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
```

Comment:
Let's use regular serde_json here instead of our fork. This was necessary before we allowed floats.

Reply:
I tried it with serde_json v1.0.40 (https://github.com/nearprotocol/json?rev=1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331#1f5779f3). It failed with gas instrumentation failure. I am filing an issue in near-bindgen.

Reply:
https://github.com/nearprotocol/near-bindgen/issues/64

Reply:
Let's hold off with this change until testnet is updated. https://github.com/nearprotocol/create-near-app/issues/75 reminder issue